nimbus-ui/nimbusui/src/app/components/platform/form-element.component.html

```diff
@@ -2,6 +2,7 @@
 <div [formGroup]="form" *ngIf="element.config?.uiStyles?.attributes?.dataEntryField" [hidden]="!element?.visible" [ngClass]="getComponentClass()">
    
     <nm-input id="{{id}}" [formControlName]="element.config?.code" *ngIf="element.config?.uiStyles?.attributes?.type==componentTypes.text.toString()" [element]="element" [form]="form"></nm-input>
+    <nm-mask id="{{id}}" [formControlName]="element.config?.code" *ngIf="element.config?.uiStyles?.attributes?.alias==componentTypes.mask.toString()" [element]="element" [form]="form"></nm-mask>
```

Comment:
1. Please add to `ViewConfig.Form` that `@InputMask` is an allowed child component.